src/awsv4.rs

```diff
@@ -1,5 +1,5 @@
 use chrono::DateTime;
-use chrono::Utc;
+pub use chrono::Utc;
```

Comment:
I would prefer to keep the `use`, and import `chrono::Utc` directly in other modules that might need it.